examples/time_frequency/plot_stockwell.py

```diff
@@ -45,6 +45,6 @@
 power, itc = tfr_stockwell(epochs, fmin=6., fmax=30., decim=4, n_jobs=1,
                            width=.3, return_itc=True)
 
-power.plot([0], baseline=(-0.5, 0), mode=None, title='S-transform (power)')
+power.plot([0], baseline=None, mode=None, title='S-transform (power)')
```

Comment:
From the long discussion before, it wasn't doing anything before, this
makes it more explicit (mode=None does nothing) and avoids what is now an
error (mode=None with a baseline not None)